otter/convergence/service.py

```diff
@@ -183,6 +185,13 @@ def execute_convergence(tenant_id, group_id, build_timeout,
                                            status=ScalingGroupStatus.ACTIVE))
             yield cf_msg('group-status-active',
                          status=ScalingGroupStatus.ACTIVE.name)
+        else:
+            # Clear servers cache and update it with latest servers
```

Comment:
Related to my previous comment in `get_scaling_group_servers`, I'm not sure why we're waiting until here to update the servers cache, instead of doing it in the gathering process, since these servers we're saving here are exactly the ones we gathered.